pkg/httpstream/spdy/stream.go

```diff
@@ -0,0 +1,45 @@
+package spdy
+
+import "github.com/docker/spdystream"
+
+// stream is a wrapper around spdystream.Stream that adds some additional
+// behavior to the Close() and Reset() methods.
+type stream struct {
```

Comment:
I originally had more logic in there. I'm going to try taking it out and see if it's needed any more.